tests/conftest.py

```diff
@@ -25,16 +27,65 @@ def app_ctx(app: Flask) -> t.Generator[AppContext, None, None]:
         yield ctx
 
 
-@pytest.fixture
-def db(app: Flask) -> SQLAlchemy:
-    return SQLAlchemy(app)
+test_classes = [
```

Comment:
The tests could run across both sqlalchemy 1 and 2, if I use importlib.metadata.version, so only use [None] if its <2 otherwise use 2.

Reply:
No longer relevant, since we're keeping the >=2 bound.